app/_locales/en/messages.json

```diff
@@ -820,15 +820,18 @@
   "editGasLow": {
     "message": "Low"
   },
+  "editGasMaxBaseFeeGWEIImbalance": {
+    "message": "Max base fee cannot be lower than priority fee"
+  },
   "editGasMaxBaseFeeHigh": {
     "message": "Max base fee is higher than necessary"
   },
-  "editGasMaxBaseFeeImbalance": {
-    "message": "Max base fee cannot be lower than priority fee"
-  },
   "editGasMaxBaseFeeLow": {
     "message": "Max base fee is low for current network conditions"
   },
+  "editGasMaxBaseFeeMultiplierImbalance": {
+    "message": "Multiplier is low relative to \"Priority fee\""
+  },
```

Comment:
@darkwing  : this is more intuitive message

RefL https://github.com/MetaMask/metamask-extension/pull/13013#issuecomment-989179553